build/InstallAndroid.ps1

```diff
@@ -3,6 +3,7 @@ $fileName = "AndroidTools.zip"
 $source = "C:\Downloads\$fileName"
 $destination = "C:\Downloads\AndroidSdkTools"
 $androidSdk = "C:\Program Files (x86)\Android\android-sdk"
+$androidSdkVersion28 = "platforms;android-28"
```

Comment:
Why do we need this older version? Can we just use 30?

Reply:
Our SDK uses 28 and it was removed from the OneBranch hosted agent VM. we can try removing this SDK from MSAL in another PR